Editor/TestTools/EditorImguiTester/EditorImguiTest.cs

```diff
@@ -178,11 +192,12 @@ private void BaseWhen(EditorWindow window, EditorWindowTestPlayer.FinishedHandle
             TextAsset recordedActionsAsset = AssetDatabase.LoadAssetAtPath<TextAsset>(PathToRecordedActions);
             if (recordedActionsAsset == null)
             {
-                throw new UserActionsRecordNotFoundException("User actions at path {0} are not found.", PathToRecordedActions);
+                throw new UserActionsRecordNotFoundException("User actions at path {0} are not found.",
+                    PathToRecordedActions);
```

Comment:
Is this jump intended? 🤔

Reply:
No. Still configuring rider at my new setup.